cheroot/ssl/builtin.py

```diff
@@ -23,17 +27,32 @@
     except ImportError:
         DEFAULT_BUFFER_SIZE = -1
 
+try:
+    from contextlib import suppress
+except ImportError:
+    # suppress was added in Python 3.4
```

Comment:
I think that the proper place for the shim would be the `.._compat` module.

Reply:
moved